src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -157,6 +157,10 @@ module sonic-device_metadata {
                     type string;
                 }
 
+                leaf mgmt_type {
```

Comment:
What does it mean? It is explained in HLD?

Reply:
I can add a description field, if this helps.

Reply:
done